src/formatters/__tests__/number.test.ts

```diff
@@ -0,0 +1,11 @@
+import { formatNumberWithCommas } from '@formatters/number';
+
+describe('number formatters', () => {
```

Comment:
As you are using something that is a built in solution that we know it works, this test I will found it overkill.
_It is great that you try to take advantage of our test suite tho_

Reply:
Make sense to me, I will delete the test